docs/hstaxe/installing.rst

```diff
@@ -3,23 +3,83 @@
 Installing hstaxe
 =================
 
+.. note::
+   As of current testing, HSTaXe does not support Windows. Please use our
+   legacy instructions for attempting to use HSTaXe on Windows
+
+Preparing Your Local Environment
+--------------------------------
+We recommend using Anaconda to manage your ``hstaxe`` environment.
+
+You may want to consider installing ``hstaxe`` in a new virtual or conda
+environment to avoid version conflicts with other packages you may have
+installed.
+
+Start by creating an empty conda environment:
+
+.. code-block:: bash
+
+    conda create --name hstaxe-env "python>=3.8, <3.11"
+    conda activate hstaxe-env
+
+Build Prerequisites
+^^^^^^^^^^^^^^^^^^^
+Because the core modules of ``hstaxe`` are written in C, we require some
+additional prerequisites to be installed before attempting to install ``hstaxe``:
+
+.. code-block:: bash
+    
+    conda install gsl cfitsio make automake autoconf libtool pkg-config -y
+    conda install wcstools -c https://conda.anaconda.org/conda-forge/ --override-channels -y
+
+
```

Comment:
```suggestion
Some architectures (for example systems with Apple M1 processors) may not have a
wheel available for ``hdf5``. This can manifest as a failed ``tables`` install and can be
solved by running the following before repeating the failed install step:

.. code-block:: bash

    conda install hdf5 -c conda-forge
    
```
Added a note about the `HDF5` problem.

Reply:
This isn't quite right.  The issue is that when a `tables` wheel is not available, it requires `hdf5` (the C package; not the `h5py` python package) to be installed (as a build-time dependency) to build `tables` from source.

Reply:
Edited, thanks for clarifying.

Reply:
Looks good, thanks!